src/recovery/cubic.rs

```diff
@@ -463,32 +476,32 @@ mod tests {
         let now = Instant::now();
         let prev_cwnd = r.cwnd();
 
-        // Send initcwnd full MSS packets to become no longer app limited
-        for _ in 0..recovery::INITIAL_WINDOW_PACKETS {
-            r.on_packet_sent_cc(r.max_datagram_size, now);
-        }
-
         // Trigger congestion event to update ssthresh
         r.congestion_event(now, packet::EPOCH_APPLICATION, now);
 
         // After congestion event, cwnd will be reduced.
         let cur_cwnd = (prev_cwnd as f64 * BETA_CUBIC) as usize;
         assert_eq!(r.cwnd(), cur_cwnd);
 
+        // Shift current time to 1 RTT
```

Comment:
```suggestion
        // Shift current time by 1 RTT.
```

Just an english language thing, unless you really mean that you want to shift to 100ms after the big bang :D